.ebextensions/nginx.config

```diff
@@ -0,0 +1,8 @@
+# From http://stackoverflow.com/a/18951706
+files:
+    "/etc/nginx/conf.d/proxy.conf" :
```

Comment:
Okay, I'm not familiar with configuring nginx. I was just copying and pasting as you suggested.